tests/test_permissions.py

```diff
@@ -468,3 +472,30 @@ def test_permission_denied_for_object_with_custom_detail(self):
             detail = response.data.get('detail')
             self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
             self.assertEqual(detail, self.custom_message)
+
+
+class CheckAttributeError(AttributeError):
+    pass
+
+
+class GetQuerysetRaises(generics.ListAPIView):
+    def get_queryset(self):
+        raise CheckAttributeError("Something terrible occurred deep down in the call stack")
```

Comment:
I subclassed `AttributeError` for the test for the purpose of being able to check for that exact exception being propagated up, which was the point of the patch in the first place :)